DeployJenkinsfile

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env groovy
 def artifactoryServer = Artifactory.server('Digital-Artifactory')
 
-library "security-ci-commons-shared-lib"
-def NODE = nodeDetails("uaa-upgrade")
+def NODE = [LABEL: "dind", IMAGE: "dig-grid-artifactory.apps.ge.com/pgog-fss-iam-uaa-docker-stage/uaa-ci-testing:latest",
```

Comment:
Lets specify node details inline in the agent section. Pipeline will read easier

Reply:
Done